nupic/frameworks/opf/expdescriptionapi.py

```diff
@@ -186,6 +197,31 @@ def __validateNupicControl(self, control):
 
 
   #############################################################################
+  def normalizeStreamSource(self, stream):
+    # The stream source in the task might be in several formats, so we need
+    # to make sure it gets converted into an absolute path:
+    source = stream['source'][len(FILE_SCHEME):]
+    # If the source is already an absolute path, we won't use pkg_resources.
+    if os.path.isabs(source):
+      absPath = source
+    else:
+      absPath = resource_filename("examples.prediction.data", source)
+    stream['source'] = FILE_SCHEME + absPath
+
+
+  #############################################################################
+  def normalizeStreamSources(self):
```

Comment:
The docstring is on the abstract function at https://github.com/numenta/nupic/pull/1902/files#diff-f18f19195652034cb22863b889dba57bR102